lib/src/database/indexeddb_box.dart

```diff
@@ -83,8 +83,14 @@ class BoxCollection with ZoneTransactionMixin {
     return _db.close();
   }
 
+  @Deprecated('use collection.deleteDatabase now')
   static Future<void> delete(String path, [dynamic factory]) =>
       (factory ?? window.indexedDB!).deleteDatabase(path);
+
+  Future<void> deleteDatabase(String name, [dynamic factory]) async {
+    await close();
+    await (factory ?? window.indexedDB).deleteDatabase(name);
+  }
```

Comment:
what's the difference between path and name?

Reply:
none, indexedDb just doesn't have a path so I wanted to avoid any confusion here https://developer.mozilla.org/en-US/docs/Web/API/IDBFactory/deleteDatabase

Reply:
Then why not rename the parameter for delete?

Reply:
done, altho that's deprecated already!